src/components/CardList/CardList.tsx

```diff
@@ -16,7 +22,11 @@ function CardList({ items, ...rest }) {
       <ul className={styles.list}>
         {items.map((item) => (
           <li key={item.id}>
-            <Card item={item} {...rest} onClick={() => handleClick(`${item.url}`)} />
+            <Card
+              item={item}
+              {...rest}
+              onClick={() => handleClick(`${item.url}`)}
```

Comment:
템플릿 리터럴없이 이렇게 표현해줘도 될 것 같아요~
```suggestion
              item={item}
              {...rest}
              onClick={() => handleClick(item.url)}
```